src/proxy-ui-api/frontend/src/views/CertificateDetails/CertificateDetails.vue

```diff
@@ -1,17 +1,23 @@
 <template>
   <div class="wrapper xrd-view-common">
-    <div class="new-content">
+        <div class="new-content">
       <subViewTitle :title="$t('cert.certificate')" @close="close" />
-      <template v-if="certificate">
-        <div class="cert-hash-wrapper">
-          <certificateHash :hash="certificate.hash" />
+      <div class="details-view-tools" v-if="certificate">
+        <large-button v-if="!certificate.active" outlined @click="activateCertificate(certificate.certificate_details.hash)">{{$t('action.activate')}}</large-button>
```

Comment:
See comment on `TokenCertificateService`. It is not enough to check if !certificate.active, you need to also check if certificate is saved to configuration.

Reply:
will be handled by https://jira.niis.org/browse/XRDDEV-815